tests/CMakeLists.txt

```diff
@@ -62,5 +62,15 @@ add_test(NAME adv-tests
   WORKING_DIRECTORY "${CMAKE_CURRENT_LIST_DIR}/.."
 )
 
+if(maat_BUILD_PYTHON_BINDINGS AND maat_RUN_PYTHON_TESTS)
```

Comment:
`maat_RUN_PYTHON_TESTS` is enabled by default and disabled for the `sanitize` CI job